### PR TITLE
dev-deploy usage restricted on mainnet

### LINKED ISSUE #657: Dev-deploy on MainNet shows unclear error
**Describe the bug**
When running dev-deploy with MainNet network selected, the error is unclear

**To Reproduce**
1. export NEAR_ENV=mainnet
2. go to some contract
3. 
```
$ near dev-deploy --wasmFile=res/fungible_token.wasm
An error occured
NotFoundError: Not Found
    at /usr/local/lib/node_modules/near-cli/node_modules/near-api-js/lib/utils/web.js:57:44
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
    at async Object.exponentialBackoff [as default] (/usr/local/lib/node_modules/near-cli/node_modules/near-api-js/lib/utils/exponential-backoff.js:7:24)
    at async Object.fetchJson (/usr/local/lib/node_modules/near-cli/node_modules/near-api-js/lib/utils/web.js:45:22)
    at async UrlAccountCreator.createAccount (/usr/local/lib/node_modules/near-cli/node_modules/near-api-js/lib/account_creator.js:41:9)
    at async createDevAccountIfNeeded (/usr/local/lib/node_modules/near-cli/commands/dev-deploy.js:90:5)
    at async devDeploy (/usr/local/lib/node_modules/near-cli/commands/dev-deploy.js:45:23)
    at async Object.handler (/usr/local/lib/node_modules/near-cli/utils/exit-on-error.js:39:9) {
  [stack]: 'NotFoundError: Not Found\n' +
    '    at /usr/local/lib/node_modules/near-cli/node_modules/near-api-js/lib/utils/web.js:57:44\n' +
    '    at processTicksAndRejections (internal/process/task_queues.js:94:5)\n' +
    '    at async Object.exponentialBackoff [as default] (/usr/local/lib/node_modules/near-cli/node_modules/near-api-js/lib/utils/exponential-backoff.js:7:24)\n' +
    '    at async Object.fetchJson (/usr/local/lib/node_modules/near-cli/node_modules/near-api-js/lib/utils/web.js:45:22)\n' +
    '    at async UrlAccountCreator.createAccount (/usr/local/lib/node_modules/near-cli/node_modules/near-api-js/lib/account_creator.js:41:9)\n' +
    '    at async createDevAccountIfNeeded (/usr/local/lib/node_modules/near-cli/commands/dev-deploy.js:90:5)\n' +
    '    at async devDeploy (/usr/local/lib/node_modules/near-cli/commands/dev-deploy.js:45:23)\n' +
    '    at async Object.handler (/usr/local/lib/node_modules/near-cli/utils/exit-on-error.js:39:9)',
  message: 'Not Found',
  [name]: 'NotFoundError'
}
```

**Expected behavior**
Show error in the form `Network ${network} doesn't support dev-deploy. Use export NEAR_ENV=testnet to switch to TestNet` or similar.

### PR DESCRIPTION
**Pull request by volovyk-s**
closes #657

### CODE REVIEW

File: commands/dev-deploy.js [hunk 1]
```diff
@@ -36,6 +36,10 @@ module.exports = {
 };
 
 async function devDeploy(options) {
+    const currentNetwork = options.networkId;
```
Comment (vgrichina) [inline commands/dev-deploy.js:39]: Better test for mainnet specifically, betanet should support dev-deploy for example

Comment (volovyk-s) [reply commands/dev-deploy.js:39]: @vgrichina
Yes, I thought the same, but it does not work on `betanet`, and `dev-deploy` docs are explicitly targeting `testnet`.

Comment (vgrichina) [reply commands/dev-deploy.js:39]: @volovyk-s why it doesn't work on betanet, what does happen?

Comment (volovyk-s) [reply commands/dev-deploy.js:39]: @vgrichina It throws an error:
`BadRequestError: Error: {"index":0,"account_id":"dev-1618909283091-6810991","predecessor_id":"betanet","registrar_account_id":"near","kind":{"account_id":"dev-1618909283091-6810991","registrar_account_id":"near","predecessor_id":"betanet"}}
`

Comment (vgrichina) [reply commands/dev-deploy.js:39]: @volovyk-s so this seems a betanet helper misconfiguration issue that we need to fix

Comment (vgrichina) [reply commands/dev-deploy.js:39]: Seems to work fine for me when sending request directly:

```
➜  near-contract-helper git:(master)  http post https://helper.betanet.near.org/account newAccountId="test1.betanet" newAccountPublicKey="ed25519:JCdUG8doFbVHTCtxuy2RVXraHg8NRhxdCcpiyGgcBMAh"
HTTP/1.1 200 OK
Content-Encoding: gzip
Content-Length: 748
Content-Type: application/json; charset=utf-8
Date: Wed, 21 Apr 2021 05:27:58 GMT
Server: Render
Vary: Accept-Encoding
X-Ratelimit-Limit: 10
X-Ratelimit-Remaining: 9
X-Ratelimit-Reset: 1618983776.276166

{
    "receipts_outcome": [
        {
            "block_hash": "CgHd9r2d4nkMaDWfQLNzsZe2rpxGJCz9MpTVyKQTPwoq",
            "id": "8ffLR9kvx9bGjyhbHw7ogqoVwjsngXxmFaXnztCeWJwE",
            "outcome": {
                "executor_id": "test1.betanet",
                "gas_burnt": 937144500000,
                "logs": [],
                "receipt_ids": [
                    "EaeLPpkGyicTNzE5Yy3WBXfYBCXDJUXvn2gueBsYDSno"
                ],
                "status": {
                    "SuccessValue": ""
                },
                "tokens_burnt": "93714450000000000"
            },
            "proof": []
        },
        {
            "block_hash": "EQcvbmkFtDFRwuPZdrAqGHX7946ti1CDvwuGCsPi6tSu",
            "id": "EaeLPpkGyicTNzE5Yy3WBXfYBCXDJUXvn2gueBsYDSno",
            "outcome": {
                "executor_id": "betanet",
                "gas_burnt": 0,
                "logs": [],
                "receipt_ids": [],
                "status": {
                    "SuccessValue": ""
                },
                "tokens_burnt": "0"
            },
            "proof": []
        }
    ],
    "status": {
        "SuccessValue": ""
    },
    "transaction": {
        "actions": [
            "CreateAccount",
            {
                "Transfer": {
                    "deposit": "100000001000000000000000000"
                }
            },
            {
                "AddKey": {
                    "access_key": {
                        "nonce": 0,
                        "permission": "FullAccess"
                    },
                    "public_key": "ed25519:JCdUG8doFbVHTCtxuy2RVXraHg8NRhxdCcpiyGgcBMAh"
                }
            }
        ],
        "hash": "6YPEfWVpfmq42HeTEEyhTqDLAHy8wxYUsghq7s8LQkVc",
        "nonce": 278,
        "public_key": "ed25519:9r1FcBRdop8NeQtiFWEWHNWtDWhTdAe2YvU8h7FrW53x",
        "receiver_id": "test1.betanet",
        "signature": "ed25519:3bb35VHqbrSyYy2Hf6xwKMreQ8Fmdh96E5Er7viaK8DCTi3SohZCE2TtJ8YQKGUi4qFnutBN6RwSY7Sech98QCm1",
        "signer_id": "betanet"
    },
    "transaction_outcome": {
        "block_hash": "An99mcWwbrUZ3VW2sJod7ZuWvmz7BJbrbnFzN6fMdgNi",
        "id": "6YPEfWVpfmq42HeTEEyhTqDLAHy8wxYUsghq7s8LQkVc",
        "outcome": {
            "executor_id": "betanet",
            "gas_burnt": 937144500000,
            "logs": [],
            "receipt_ids": [
                "8ffLR9kvx9bGjyhbHw7ogqoVwjsngXxmFaXnztCeWJwE"
            ],
            "status": {
                "SuccessReceiptId": "8ffLR9kvx9bGjyhbHw7ogqoVwjsngXxmFaXnztCeWJwE"
            },
            "tokens_burnt": "93714450000000000"
        },
        "proof": []
    }
}
```

Comment (volovyk-s) [reply commands/dev-deploy.js:39]: @vgrichina it works on `betanet` only when accountId has `.betanet` suffix. `near-cli` creates acc in a next way: `accountId = dev-${Date.now()}-${randomNumber}`. Without any suffix. Works fine for `testnet` but not for `betanet`.

Comment (frol) [reply commands/dev-deploy.js:39]: That is odd! Is testnet still misconfigured allowing non-suffixed account ids? cc @bowenwang1996

Comment (vgrichina) [reply commands/dev-deploy.js:39]: @frol why misconfigured? AFAIK even mainnet allows to create arbitrary account IDs if they are at least 32 characters long.

For example https://explorer.near.org/accounts/mikemikemikemikemikemikemikemike

Comment (volovyk-s) [reply commands/dev-deploy.js:39]: @frol where this configuration is? Who is responsible for it?

Comment (frol) [reply commands/dev-deploy.js:39]: Oh, I thought the account id is going to be shorter than 32 characters, so it depends on how big the `randomNumber` is then.

Comment (volovyk-s) [reply commands/dev-deploy.js]: @vgrichina @frol I have increased dev acc length.

**Review by frol — APPROVED** [review]
Does it work on betanet now?

**Comment by volovyk-s** [comment]
Yes, this is why I have changed the length of the acc.

**Pull request merged**